packages/discord.js/src/index.js

```diff
@@ -146,6 +146,9 @@ exports.GuildScheduledEvent = require('./structures/GuildScheduledEvent').GuildS
 exports.GuildTemplate = require('./structures/GuildTemplate');
 exports.Integration = require('./structures/Integration');
 exports.IntegrationApplication = require('./structures/IntegrationApplication');
+exports.InteractionCallback = require('./structures/InteractionCallback');
+exports.InteractionCallbackResource = require('./structures/InteractionCallbackResource');
+exports.InteractionCallbackResponse = require('./structures/InteractionCallbackResponse');
```

Comment:
You need to either
- use module.exports in the files so you can do exports.<clazz> here
- append the class name at the end of the require's so they're exported correctly